application/src/main/java/org/togetherjava/tjbot/features/chaptgpt/ChatGPTCommand.java

```diff
@@ -0,0 +1,50 @@
+package org.togetherjava.tjbot.features.chaptgpt;
+
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.features.CommandVisibility;
+import org.togetherjava.tjbot.features.SlashCommandAdapter;
+
+import java.util.Objects;
+
+/**
+ * <p>
+ * The implemented command is {@code /chatgpt}, upon which the bot will respond with a response
+ * generated by ChatGPT.
+ */
+public final class ChatGPTCommand extends SlashCommandAdapter {
+    private static final Logger logger = LoggerFactory.getLogger(ChatGPTCommand.class);
+    private static final String PROMPT = "prompt";
+
+    /**
+     * Creates an instance of the chatgpt command.
+     */
+    public ChatGPTCommand() {
+        super("chatgpt", "User enters a prompt and then gets a response generated by ChatGPT",
+                CommandVisibility.GUILD);
+
+        getData().addOption(OptionType.STRING, PROMPT, "Prompt sent to ChatGPT", true);
+    }
+
+    /**
+     * When triggered with {@code /chatgpt}, the bot will respond with a ChatGPT generated response.
+     *
+     * @param event the corresponding event
+     */
+    @Override
+    public void onSlashCommand(@NotNull SlashCommandInteractionEvent event) {
```

Comment:
notnull annotation is applied by default
```suggestion
    public void onSlashCommand(SlashCommandInteractionEvent event) {
```